sdk/core/azure-core/test/ut/proxy_tests/localproxy.passwd/proxypasswd

```diff
@@ -0,0 +1 @@
+user:$apr1$z0tE.mEW$6OXoAgyTD5vBkM5ON3R9h/
```

Comment:
I wonder if this type of change would get flagged by tools like CredScan.

Reply:
No clue. Nightly CI passes though, and credscan should be run on every pull request.